src/types/DateTime.ts

```diff
@@ -0,0 +1,25 @@
+import * as TypeUtils from "../TypeUtils"
+import * as _ from 'lodash'
+
+export const DateTime: SBVRType<InternalDate, NullableDate> = {
+	types: {
+		postgres: 'TIMESTAMP',
+		mysql: 'TIMESTAMP',
+		websql: 'INTEGER',
+		odata: {
+			name: 'Edm.DateTime'
+		},
+	},
+
+	fetchProcessing: (data, callback) => {
+		let processedValue: Date
+		if(_.isDate(data) || data === null) {
```

Comment:
We previously checked `data == null`, as it stands now we'll treat `undefined` differently to before